care/facility/migrations/0438_alter_dailyround_patient_category_and_more.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 4.2.10 on 2024-05-21 12:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    def rename_categories_in_events(apps, schema_editor):
+        PatientConsultationEvent = apps.get_model(
+            "facility", "PatientConsultationEvent"
+        )
+
+        PatientConsultationEvent.objects.filter(
+            event_type__name="CATEGORY", value__category="Stable"
+        ).update(value={"category": "Mild"})
+        PatientConsultationEvent.objects.filter(
+            event_type__name="PATIENT_CATEGORY", value__category="Stable"
+        ).update(value={"patient_category": "Mild"})
+        PatientConsultationEvent.objects.filter(
+            event_type__name="CATEGORY", value__category="Abnormal"
+        ).update(value={"category": "Moderate"})
+        PatientConsultationEvent.objects.filter(
+            event_type__name="PATIENT_CATEGORY", value__category="Abnormal"
+        ).update(value={"patient_category": "Moderate"})
```

Comment:
note: this is just to change the labels, as we serialize the labels while creating events